app/services/workspace.service.ts

```diff
@@ -0,0 +1,57 @@
+import { Injectable } from "@angular/core";
+import { List } from "immutable";
+import { BehaviorSubject, Observable } from "rxjs";
+
+// tslint:disable:no-var-requires max-line-length
+const stripJsonComments = require("strip-json-comments");
+
+import { Workspace } from "app/models";
+
+// TODO: can we just read all JSON files in a directory?
+const adminWorkspace = JSON.parse(stripJsonComments(require("app/components/workspace/json-templates/admin-workspace.json")));
```

Comment:
Let's start like that, we'll look into custom user workspace later. You can also remove the stripJsonComments here. This file shouldn't have some